generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/FormattedAs.feature

```diff
@@ -288,9 +289,10 @@ Feature: User can specify that a value is so formatted
       | 1                        | null   |
       | 2018-02-01T16:17:18.199Z | null   |
 
-  Scenario: Running an 'formattedAs' request with a null value should not attempt to format the data
+  @ignore #266, formattedAs %b doesn't work correctly with null values
+  Scenario: Running an 'formattedAs' request with a null value should return false
```

Comment:
Not a fan of this, personally - if I had a non-mandatory field and I specified a format for it, I'd be surprised if that format applied in the case where the field isn't populated. I'd bypass formatting entirely for non-values.

Reply:
Ok, we have a ticket for this #266 where we can cover this change. It was originally part of this PR but removed to allow a focus on the changes being made.

Reply:
Ok, we have a ticket for this #266 where we can cover this change. It was originally part of this PR but removed to allow a focus on the changes being made.

Reply:
When I say change, I mean updating the scenario/s